apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/index.tsx

```diff
@@ -163,15 +163,6 @@ export default function DocumentEditor({
   })
   const { metadata, runReadMetadata } = useMetadata({ onMetadataProcessed })
 
-  useEffect(() => {
-    runReadMetadata({
-      prompt: value,
-      documents: _documents,
-      document,
-      fullPath: document.path,
-    })
-  }, [])
```

Comment:
yeah not sure why this isn't reacting to changes in its arguments with some healthy debouncing